config/epochConfig.go

```diff
@@ -87,6 +87,7 @@ type EnableEpochs struct {
 	FixAsyncCallBackArgsListEnableEpoch               uint32
 	FixOldTokenLiquidityEnableEpoch                   uint32
 	RuntimeMemStoreLimitEnableEpoch                   uint32
+	RuntimeCodeSizeFixEnableEpoch                     uint32
```

Comment:
field not defined in the enableEpoch.toml. Also, should include this new flag in the config_test.go tests

Reply:
Added.